crates/cast/src/opts.rs

```diff
@@ -496,6 +496,14 @@ pub enum CastSubcommand {
 
         #[command(flatten)]
         rpc: RpcOpts,
+
+        /// If specified, the transaction will be converted to a TransactionRequest.
+        #[arg(long)]
+        to_request: bool,
+
+        /// Output the TransactionRequest in JSON format.
+        #[arg(long, requires = "to_request")]
+        json: bool,
```

Comment:
I think we can unify these to, so that to_request always behaves like "print the request as json" because without json it's basically the same as just cast tx